src/renderer/rendered_window.rs

```diff
@@ -238,7 +242,7 @@ impl RenderedWindow {
         paint.set_color(default_background.with_a(a));
         root_canvas.draw_rect(pixel_region, &paint);
 
-        paint.set_color(Color::from_argb(a, 255, 255, 255));
```

Comment:
This broke transparent floating windows (windows which render with transparency over other windows with a blurred background)

I'm wondering if you could give some details about this change?

Reply:
IIRC this color is used for drawing text, so I set it to fully opaque, because otherwise the text is hard to read.